### PR TITLE
Add scopes to GetUserResponse type

### PR DESCRIPTION
**Pull request by cinder-fish**
e.g. https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Database-Users/operation/listDatabaseUsers

### CODE REVIEW

File: src/user.d.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export interface GetUserResponse {
     x509Type: string;
     groupId: string;
     roles: UserRoleConfig[];
+    scopes: UserScopeConfig[];
```
Comment (montumodi) [inline src/user.d.ts:13]: Should this be nullable or will it always be returned?

Comment (cinder-fish) [reply src/user.d.ts:13]: I believe always, but will double-check

(Certainly optional on create/update)

Comment (cinder-fish) [reply src/user.d.ts:13]: Created a user without scopes meaning

> If omitted, MongoDB Cloud grants the database user access to all the clusters and MongoDB Atlas Data Lakes in the project.

And got back a default cluster-global scope, 
```json
{
...
"scopes": [
    {
      "name": "test",
      "type": "CLUSTER"
    }
  ],
...
}
```
(`name` here is the name of the cluster)

So yep, shoud always be returned

Comment (montumodi) [reply src/user.d.ts:13]: Perfect. Good to go. I will merge and publish.

Comment (montumodi) [reply src/user.d.ts:13]: Published new version - https://www.npmjs.com/package/mongodb-atlas-api-client/v/3.16.0

**Pull request merged**